src/libcore/ptr/mod.rs

```diff
@@ -92,6 +92,12 @@ mod unique;
 #[unstable(feature = "ptr_internals", issue = "0")]
 pub use unique::Unique;
 
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Hm, these should be entirely invisible to end-users -- do we lint right now for the stability on the (private) module?

If we do need a stability annotation let's mark them as `ptr_internals` (like Unique above).

Reply:
Checked it again, we do NOT need those annotations for private modules.
I added those line while I had both modules as *pub* and the tidy check required the stability annotations.
I'll remove both of them.